src/org/usfirst/frc/team2399/robot/OI.java

```diff
@@ -1,6 +1,11 @@
 package org.usfirst.frc.team2399.robot;
 
+import org.usfirst.frc.team2399.robot.commands.JoyShoot;
+import org.usfirst.frc.team2399.robot.commands.ShooterStop;
```

Comment:
It was Lauren's executive decision to rename these after merge, we will establish naming conventions in a journal tonight.